mailbox/api/src/main/java/org/apache/james/mailbox/model/SimpleMailboxACL.java

```diff
@@ -951,29 +869,29 @@ public SimpleMailboxACL(Properties props) throws UnsupportedRightException {
 
         Map<MailboxACLEntryKey, MailboxACLRights> m = new HashMap<>(props.size() + props.size() / 2 + 1);
 
-        if (props != null) {
```

Comment:
you removed null props support ?

Reply:
Intellij reported it to be never null

Reply:
just before we do props.size() so I guess props is expected to be not null :)

Reply:
Commit for this added to the refactoring PR.